boto/ec2/elb/__init__.py

```diff
@@ -756,3 +757,71 @@ def detach_lb_from_subnets(self, name, subnets):
                                'Subnets.member.%d')
         return self.get_list('DetachLoadBalancerFromSubnets',
                              params, None)
+
+    # Tag methods
+
+    def _build_tag_param_list(self, params, tags):
```

Comment:
The name and implementation come pretty directly from `ec2/connection.py`, around line 4137.  (Which isn't to say that the EC2 implementation shouldn't change as well!)  Unlike in EC2, though, the ELB version is only ever called once... so it could simply be inlined into the calling function.

Are you imagining that this function would return a new dict containing _only_ the tag key/values, and then the caller would `update()` the actual params dict with the returned one?